app/controllers/items_controller.rb

```diff
@@ -6,14 +7,19 @@ def index
 
     @items = if @keyword.present?
       RWS::Ichiba::Item.
-        search(keyword: params[:keyword], imageFlag: 1).first(10)
+        search(keyword: params[:keyword], genre_id: @genre.id, imageFlag: 1).first(10)
```

Comment:
__[rubocop]__
* Line is too long. [86/80]

<!-- Comment by [SideCI](https://sideci.com) -->